deploy-manage/security/self-setup.md

```diff
@@ -0,0 +1,56 @@
+---
+navigation_title: "Self-managed security setup"
+applies_to:
+  deployment:
+    self: ga
+---
+
+% scope: initial security setup in self-managed deployments, following the automatic or manual (minimal, basic, https) procedures
+# Set up security in self-managed deployments
+
+This section explains how to perform the initial security setup for self-managed deployments, including configuring TLS certificates to secure {{es}} and {{kib}} endpoints, setting passwords for built-in users, and generating enrollment tokens to connect {{kib}} or additional {{es}} nodes to the cluster.
+
+Self-managed deployments support two approaches for the initial setup: automatic and manual. Note that securing {{kib}} always requires some manual configuration.
+
+For guidance on configuring additional security features, refer to [](./secure-your-cluster-deployment.md).
+
+## Automatic configuration [automatic-configuration]
+
+Since version 8.0, {{es}} automatically enables security features on first startup when the node is not part of an existing cluster and none of the [incompatible settings](./self-auto-setup.md#stack-existing-settings-detected) have been explicitly configured.
+
+The automatic configuration:
+
+* Generates TLS certificates for the [transport and HTTP layers](./secure-cluster-communications.md#communication-channels)
+* Applies TLS configuration settings to `elasticsearch.yml`
+* Sets a password for the `elastic` superuser
+* Creates an enrollment token to securely connect {{kib}} to {{es}}
+
+This automatic setup is the quickest way to get started and ensures your cluster is protected by default.
+
+::::{note}
+The automatic configuration does not enable TLS on the {{kib}} HTTP endpoint. To encrypt browser traffic to {{kib}}, follow the steps in [](./set-up-basic-security-plus-https.md#encrypt-kibana-browser).
+::::
```

Comment:
I think we want this info here too:

https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/941/deploy-manage/security/secure-cluster-communications#encrypt-http-communication

Reply:
Added in the self-managed tab.